### PR TITLE
Design QA updates

### PR DESCRIPTION
**Pull request by reenybeeny**

### CODE REVIEW

File: styleguide/source/_patterns/03-organisms/by-author/action-finder.twig [hunk 1]
```diff
@@ -31,9 +31,9 @@
           {% endif %}
         </li>
       {% endfor %}
-    </ul>    
-    <div class="ma__action-finder__category">All Actions:</div>
-    <ul class="ma__action-finder__items ma__action-finder__items--all">
+    </ul>
+    <div class="ma__action-finder__category">All Actions & Guides:</div>
+    <ul class="ma__action-finder__items">
```
Comment (jasonstanbery) [inline styleguide/source/_patterns/03-organisms/by-author/action-finder.twig]: @reenybeeny are you aware (and is it on purpose) that you're removing the `ma__action-finder__items--all` class from line 36? That just went in this afternoon so I want to make sure it's on-purpose.

**Review by jasonstanbery — COMMENTED** [review]
Just wondering about this one thing.

File: styleguide/source/_patterns/03-organisms/by-author/action-finder.twig (hunk 1 above)
Comment (reenybeeny) [reply styleguide/source/_patterns/03-organisms/by-author/action-finder.twig]: @jasonstanbery - I think that was a mistake on my part. We can revert that.

**Comment by jasonstanbery** [comment]
Closing this, I fixed the one line and re-submitted the PR.

https://github.com/massgov/mayflower/pull/100

**Pull request closed without merging**